vignettes/forcats.Rmd

```diff
@@ -44,16 +44,55 @@ To do so, we can use the function `fct_infreq()`:
 
 ```{r fct-infreq-hair}
 #| fig.alt: >
-#|   The A bar chart of hair color, now ordered so that the least
+#|   The A bar chart of hair color, now ordered so that generally the least
 #|   frequent colours come first and the most frequent colors come last.
 #|   This makes it easy to see that the most common hair color is none 
-#|   (~35), followed by brown (~18), then black (~12).
-ggplot(starwars, aes(x = fct_infreq(hair_color))) + 
+#|   (~35), followed by brown (~18), then black (~12). Surprisingly,
+#|   NAs are at the top of the graph, even though there are ~5 NAs and other
+#|   colors have smaller values.
+ggplot(starwars, aes(y = fct_infreq(hair_color))) + 
   geom_bar() + 
-  coord_flip()
+  labs(y = "Hair color")
 ```
 
-Note that `fct_infreq()` it automatically puts NA at the top, even though that doesn't have the smallest number of entries.
+It's a little surprising that the `NA` bar isn't ordered by frequency.
+To understand why, we need to make a brief digression to discuss implicit vs explicit `NA`s in factors.
+
+## Implicit vs explicit missing values
```

Comment:
```suggestion
## Implicit vs explicit missing levels
```

Is it about the level or the value in the data?

In the implicit case, there is actually an `NA` _value_ in the data. i.e. it isn't represented by an integer code in that case, which confused me a little

Reply:
Yeah, I think that makes sense